core/node/events/miniblock_producer.go

```diff
@@ -276,7 +276,10 @@ func (p *miniblockProducer) TestMakeMiniblock(
 
 		err = SleepWithContext(ctx, 10*time.Millisecond)
 		if err != nil {
-			return nil, err
+			return nil, AsRiverError(err, Err_INTERNAL).
```

Comment:
As part of this PR I added some more tags to errors that popped up in testing in order to track down race conditions, and I left them in since they may be helpful in the future.